### PR TITLE
update Terminus install, prettify commands.json

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes #

## Summary

**[Terminus Commands](https://pantheon.io/docs/terminus/commands)** - Fixed the Terminus installer script to update the version referenced by Gatsby for the Terminus manual. Updates Terminus changelog and commands table. Prettified the Terminus `commands.json` so that any changes we make to it in the future are easier to track.

## Effect

The following changes are already committed:

* fix the way the CI installs terminus to retrieve commands
* install latest terminus
* prettify `commands.json` for better comparisons later

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] terminus releases: add `2.6.1` and `2.6.0` to the [changelog](https://pr-6871--pantheon-docs.my.pantheonfrontend.website/docs/terminus/updates#changelog)(link goes to preview of this pr)
- [ ] replace `--verbose` with `--v|vv|vvv` or similar if necessary

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6871](https://pr-6871--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `61` | `4` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6871` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/31547/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/31547/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6871--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by EdwardAngert** [comment]
@jmolivas @kporras07 this PR changes the way we install terminus for the Docs site because it's [currently broken in the CI](https://app.circleci.com/pipelines/github/pantheon-systems/documentation/5983/workflows/8a554da0-cde8-4f9f-b0fd-82caf76b4347/jobs/31568?invite=true#step-104-7)
while I was at it, I updated the version and references to it
but then it was hard to see the specific changes because the `commands.json` files were all one line, so I prettified them
in [this PR](https://github.com/pantheon-systems/documentation/pull/6795/files?authenticity_token=mHpffG8isvSR46%2BN%2FR8fiVepuV4KUCDiuRjINhZBsgzWQd%2F2hqp1LoEzqEwMh5nNqH5DloxTugqzWvOigCKxhA%3D%3D&file-filters%5B%5D=.json#diff-5b6c2e901ecab5c65baf4ee8105c8db5f6f6da9f203ed5fa92ab22196baa71f4R5)

here's the [built preview of the Updates page](https://pr-6871--pantheon-docs.my.pantheonfrontend.website/docs/terminus/updates)

can you check out the changes and let me know if there's anything missing, or if there's a reason I shouldn't:

```shell
    exec("sudo chmod ugo+rwx /tmp/");
    exec("sudo chmod ugo+rwx /usr/local/bin/");
```

**Comment by kporras07** [comment]
IIRC ugo+rwx is basically 777 and that's usually not a good idea. It's a CI build so I'm unsure if there's a real risk of doing it like this but if it could be avoided that would be great. Also, do you really need to run that command in both /tmp and /usr/local/bin ?

Are you sure it was failing in CI? To me it looks like it was initially attempting a php rename and if it fails, then it's using `sudo mv` and it should have worked.

**Comment by EdwardAngert** [comment]
> IIRC ugo+rwx is basically 777 and that's usually not a good idea. It's a CI build so I'm unsure if there's a real risk of doing it like this but if it could be avoided that would be great. Also, do you really need to run that command in both /tmp and /usr/local/bin ?
> 
> Are you sure it was failing in CI? To me it looks like it was initially attempting a php rename and if it fails, then it's using `sudo mv` and it should have worked.

@kporras07 
Not sure at all! But it worked! and I think I tried doing one and not the other without success, but I'm not super familiar/great at it
I had the same thought process about it being "safe enough" in the CI. since it's already sudo mv-ing, I figured some other command just isn't getting passed the proper privilege
Happy to accept suggestions, or to leave as-is if it's ok there

+ Re slack: I'll update the EoL table

**Comment by kporras07** [comment]
My original thought is that those sudo chmod should not be needed at all. The failing link you pasted I think wasn't actually failing.

In any case, yeah, I agree CI is "safe enough" so go ahead if you want to get it merged :)

File: .circleci/scripts/terminus-installer.php [hunk 1]
```diff
@@ -38,6 +38,8 @@ function downloadTerminus($installdir, $package)
     echo("\nDownloading Terminus " . $version . " from " . $url . " to /tmp \n");
     $couldDownload = file_put_contents("/tmp/" . $package . ".phar", file_get_contents($url));
     echo("Moving to " . $installdir . "...\n");
+    exec("sudo chmod ugo+rwx /tmp/");
+    exec("sudo chmod ugo+rwx /usr/local/bin/");
```
Comment (greg-1-anderson) [inline .circleci/scripts/terminus-installer.php:42]: I think long-term maintenance would be easier if you removed the Terminus installer, and we instead used the update tool to fetch the Terminus data and create a PR in this repo when things change. It seems to me that it shouldn't be a concern of the docs CI to know what version of Terminus to use, or to install and run it.

**Review by greg-1-anderson — APPROVED** [review]

**Pull request merged**